File: it-android/src/main/assets/tests/missing-signature-methods.js
let target = Packages.org.mozilla.javascript.android.MissingSignatureMethods

assertEquals("available", target.available() + "")

if (android.os.Build.VERSION.SDK_INT >= 33) {
    assertEquals("null", target.missingParameter(null) + "")
    assertEquals(null, target.missingReturn())
}

"success"

File: it-android/src/main/java/org/mozilla/javascript/android/MissingSignatureMethods.java
package org.mozilla.javascript.android;

import android.app.LocaleManager;

public final class MissingSignatureMethods {

    private MissingSignatureMethods() {}

    public static String available() {
        return "available";
    }

    public static String missingParameter(LocaleManager localeManager) {
        return localeManager == null ? "null" : "available";
    }

    public static LocaleManager missingReturn() {
        return null;
    }
}
